assignments/closure.js

```diff
@@ -1,5 +1,13 @@
 // ==== Challenge 1: Write your own closure ====
 // Write a simple closure of your own creation.  Keep it simple!
+  
+ const myname = 'Fnu-Bharti';
```

Comment:
This is close, but if you want the Name function to be a closure, it and your myname variable need to be inside a parent function. A closure is basically when a function has access to the variables and such inside its parent function.